resources/web/style.css

```diff
@@ -1,3 +1,58 @@
+body {
+    font-family: "Roboto", arial, sans-serif;
+    color: #aaa;
+    padding: 0;
+    margin: 0;
+    background: rgb(28, 28, 36);
+}
+
+pre {
+    border-color: #666666;
+    color: #dddddd;
+    background-color: rgba(20, 20, 20, 0.5);
+    display: block;
+    font-family: monospace;
+    white-space: pre;
+    margin: 1em 0;
+}
+
+h1 {
+    color: white;
+    padding-top: 20px;
+    margin-left: 15px;
+    font-size: 26px;
+    font-family: "Roboto", arial;
+    display: inline-block;
+}
+
+option:hover {
+    background-color:#cd201f;
+    cursor: pointer;
+}
+
+table {
+    border: 0px;
+    width: 100%;
+}
+tr:nth-child(even) {
+    font-family: "Roboto", Sans-serif;
+    background: rgba(30, 30, 30, 0.3);
+    color: #aaaaaa;
+    font-size: 14px;
+}
+tr:nth-child(odd) {
+    font-family: "Roboto", Sans-serif;
+    background: rgba(0, 0, 0, 0.3);
+    color: #aaaaaa;
+    font-size: 14px;
+}
+th, td {padding: 4px;}
```

Comment:
The history marks are pretty bad on this one though. I can't see your changes really. It looks weird. Is therer a link to the full document containing your changes?

For example. I didn't even touch .container but it shows up on your changes. Im new to github, pls help 💃